pkg/profilehelpers/profile.go

```diff
@@ -0,0 +1,17 @@
+package profilehelpers
+
+import operatorv1 "github.com/openshift/api/operator/v1"
```

Comment:
also wondering, don't you have to vendor the new API with go mod?

Reply:
I was just working with it with a go workspace until the api got merged. But yeah, I'll update the vendoring when that happens